src/google/google-calendars/googlecalendarsyncadaptor.cpp

```diff
@@ -1342,6 +1342,11 @@ void GoogleCalendarSyncAdaptor::finishedRequestingRemoteEvents(int accountId, co
     foreach (const QString &updatedCalendarId, m_calendarsFinishedRequested.keys()) {
```

Comment:
m_calendarsFinishedRequested seems to be used only inside this method and never cleared. Any chance bad data could accumulate?